src/constants/labels.json

```diff
@@ -10,7 +10,8 @@
     "haveFun": "Viel Spaß beim Lernen",
     "progressDescription": "Modulen erledigt",
     "start": "Beginnen",
-    "continue": "Weitermachen",
+    "continue": "Fortsetzen",
+    "viewModules": "Module ansehen",
```

Comment:
(optional) "module" is a naming we do not use, so please rename to "viewDisciplines".